client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -163,29 +191,36 @@ class CampaignAdminList extends SilverStripeComponent {
         );
       });
 
+      const wrapperClassnames = classnames('list-group-wrapper', {
+        'list-group-wrapper--empty': listGroupItems.length === 0,
+      });
+
       // Merge into group
       accordionBlocks.push(
-        <AccordionBlock key={groupid} groupid={groupid} title={title}>
-          {listGroupItems}
-        </AccordionBlock>
+        <div className={wrapperClassnames}>
+          <AccordionBlock key={groupid} groupid={groupid} title={title}>
+            {
+              listGroupItems.length > 0
+                ? listGroupItems
+                : <p className="list-group-item">{group.noItemsText}</p>
```

Comment:
@phalkunz please note, we're following airbnb formatting standards (to an extent) so ternary conditions should break like this:
```
condition
  ? item
  : null
```
ref: https://github.com/airbnb/javascript/#comparison--nested-ternaries the better section